modules/sandboxed-containers-deploying-workloads-with-kata-runtime-cli.adoc

```diff
@@ -0,0 +1,72 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-deploying-workloads-with-kata-runtime-cli_{context}"]
+= Deploying a workload in a sandboxed container using the CLI
+
+{sandboxed-containers-first} installs the kata runtime as a secondary optional runtime on the cluster, and not as the primary runtime.
```

Comment:
Capitalize Kata?

Reply:
@amolnar-rh 
> Capitalize Kata

Actually, in that case I think it should be backticks, `kata`, because it's the name of a `RuntimeClass`

Reply:
@c3d Noted, thanks Christophe, I'll keep that in mind if I find other instances of it!